### PR TITLE
enable app_manager to delete and list app testers

### PR DESCRIPTION
**Pull request by taquitos**
app_managers should be able to list testers and also delete them.
fix for #9910

### CODE REVIEW

File: pilot/lib/pilot/tester_manager.rb [hunk 1]
```diff
@@ -223,7 +225,7 @@ def describe_tester(tester)
       rows << ["Last name", tester.last_name]
       rows << ["Email", tester.email]
 
-      if tester.groups.length > 0
+      if tester.groups.to_s.length > 0
```
Comment (taquitos) [inline pilot/lib/pilot/tester_manager.rb]: it's possible this is nil (found during testing)

File: pilot/lib/pilot/tester_manager.rb [hunk 2]
```diff
@@ -232,7 +234,7 @@ def describe_tester(tester)
         rows << ["Latest Install Date", tester.pretty_install_date]
       end
 
-      if tester.devices.length == 0
+      if tester.devices.to_so.length == 0
```
Comment (ohayon) [inline pilot/lib/pilot/tester_manager.rb]: Should this be `to_s`?

**Review by ohayon — APPROVED** [review]
Just one little comment but otherwise 💯

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Tests have failed, see below for more information.</td>
    </tr>
  </tbody>
</table>

### Tests: 

Name | File|
--- | ---|
Fastlane Fastlane::FastFile Unlock keychain Integration set default keychain | <a href='https://github.com/fastlane/fastlane/blob/904c1c2b5744829a37f876bcad9c4cbc472b4eb4/./fastlane/spec/actions_specs/unlock_keychain_spec.rb'>unlock_keychain_spec.rb</a>|


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Pull request merged**